data/sql_updates/post_totals_create_candidate_aggregates.sql

```diff
@@ -8,7 +8,8 @@ with totals as (
         receipts,
         disbursements,
         last_cash_on_hand_end_period,
-        last_debts_owed_by_committee
+        last_debts_owed_by_committee,
+        'false' as federal_funds_flag
```

Comment:
Does this need to be a string? Can we use `false as federal_funds_flag` instead? If we can, then we also shouldn't need the `cast` below.